librosa/feature/spectral.py

```diff
@@ -16,7 +16,7 @@
 from ..core.spectrum import power_to_db, _spectrogram
 from ..core.constantq import cqt, hybrid_cqt, vqt
 from ..core.pitch import estimate_tuning
-from typing import Any, Optional, Union, Collection
+from typing import Any, Literal, Optional, Union, Collection
```

Comment:
Literal needs to import from typing_extensions, not typing

Reply:
Oh, I see... think I fixed it now